README.md

```diff
@@ -187,6 +187,17 @@ poller:
   fromBlock: 20000000
 ```
 
+#### Poller Force Start Block
+From which block to start polling. Default is `0`.
+
+cmd: `--poller-force-start-block`
+env: `POLLER_FORCESTARTBLOCK`
+yaml:
+```yaml
+poller:
+  forceStartBlock: 20000000
+```
+
```

Comment:
I would either unify the names by renaming this to `poller.forceFromBlock` or make this a boolean flag to force apply the `poller.fromBlock` even if there is data in staging.

And this flag is not needed at all if we configure persistent storage for block failures. Then failure recoverer takes care of the gaps

Reply:
it is a boolean, fixing the name and description now

Reply:
renamed. let's remove it once we are confident it's not needed anymore